velox/exec/AggregateWindow.cpp

```diff
@@ -341,7 +342,34 @@ class AggregateWindowFunction : public exec::WindowFunction {
     });
 
     // Set null values for empty (non valid) frames in the output block.
-    setNullEmptyFramesResults(validRows, resultOffset, result);
+    setEmptyFramesResult(validRows, resultOffset, emptyResult_, result);
+  }
+
+  // Precompute and save the aggregate output for empty input in emptyResult_.
+  // This value is returned for rows with empty frames.
+  void computeDefaultAggregateValue(
+      const std::string& name,
```

Comment:
Doesn't seem like either name or resultType are used in the function. Please remove them.